src/rust/ide/view/graph-editor/src/component/edge.rs

```diff
@@ -1289,7 +1314,10 @@ impl EdgeModelData {
 
     /// Set the color of the edge. Also updates the focus color (which will be a dimmed version
     /// of the main color).
-    pub fn set_color(&self, color:color::Lcha) {
+    fn set_color(&self, color:color::Lcha) {
+        // We must never use alpha in edges, as it will show artifacts with overlapping sub-parts.
+        let color:color::Lcha = color.opaque.into();
+
         // FIXME : StyleWatch is unsuitable here, as it was designed as an internal tool for shape system (#795)
```

Comment:
I'm curious about one thing here. Is it using the styleWatch you've created with this PR? If so, is it still an internal tool, or this FIXME and all FIXME'S like this can be removed from all other files?

Reply:
This is not using a new `StyleWatch`. Right now, I'm using the old one, with a thin wraper for colors.